app/controllers/api/v1/profiles_controller.rb

```diff
@@ -0,0 +1,45 @@
+module Api::V1
+  class ProfilesController < ApiController
+
+    def index
+      if params[:uids].present?
+        profiles = Profile.where(uid: params[:uids])
+      else
+        profiles = Profile.all
```

Comment:
Just wondering in practice if there is ever a need to return all `profiles` at the same time?